libs/application/template-api-modules/src/lib/modules/templates/parental-leave/parental-leave.service.ts

```diff
@@ -190,6 +192,29 @@ export class ParentalLeaveService extends BaseTemplateApiService {
     }
   }
 
+  async setBirthDate({ application }: TemplateApiModuleActionProps) {
+    /*
+    MOCK
+    return {
+      dateOfBirth: '2023-02-04',
+    }
+  */
+    try {
+      const applicationInformation = await this.applicationInformationAPI.applicationGetApplicationInformation(
+        {
+          applicationId: application.id,
+        },
+      )
+      return {
+        dateOfBirth: applicationInformation.dateOfBirth,
+      }
+    } catch (e) {}
+
+    return {
```

Comment:
do something like this:
catch (e) {
    // Ignore
  }
  Otherwise you will get linting error

Reply:
@qdd1 
Thanks for catching 👍 . I forgot about this.